test/examples/make.jl

```diff
@@ -703,6 +703,59 @@ else
     nothing
 end
 
+examples_latex_simple_native_doc = if "latex_simple_native" in EXAMPLE_BUILDS
+    @info("Building mock package docs: LaTeXWriter/latex_simple_native")
+    @quietly makedocs(
+        format=Documenter.LaTeX(platform="native", version=v"1.2.3"),
+        sitename="Documenter LaTeX Simple Native",
+        root=examples_root,
+        build="builds/latex_simple_native",
+        source="src.latex_simple",
+        pages=["Main section" => ["index.md"]],
+        doctest=false,
+        debug=true,
+    )
+else
+    @info "Skipping build: LaTeXWriter/latex_simple_native"
+    @debug "Controlling variables:" EXAMPLE_BUILDS get(ENV, "DOCUMENTER_TEST_EXAMPLES", nothing)
+    nothing
+end
+
+examples_latex_simple_texify_doc = if "latex_simple_latexmk" in EXAMPLE_BUILDS
+    @info("Building mock package docs: LaTeXWriter/latex_simple_latexmk")
+    @quietly makedocs(
+        format=Documenter.LaTeX(platform="latexmk", version=v"1.2.3"),
+        sitename="Documenter LaTeX Simple LaTeXMk",
+        root=examples_root,
+        build="builds/latex_simple_latexmk",
+        source="src.latex_simple",
+        pages=["Main section" => ["index.md"]],
+        doctest=false,
+        debug=true,
+    )
+else
+    @info "Skipping build: LaTeXWriter/latex_simple_latekmk"
+    @debug "Controlling variables:" EXAMPLE_BUILDS get(ENV, "DOCUMENTER_TEST_EXAMPLES", nothing)
+    nothing
+end
```

Comment:
This pretty much duplicates `latex_simple_nondocker` (which, to be fair, I'm not 100% sure if we're running on CI right now..). So let's just make that explicitly `platform = "latexmk"`, and test the `hastex` behavior with a unit test.

Reply:
But if there is no dedicated CI for `texify` wouldn't it mean that if some options change in `texify`, which is unlikely though, that would not be caught in tests ?